daemon/lokinet-bootstrap.cpp

```diff
@@ -0,0 +1,64 @@
+#include <cpr/cpr.h>
+#include <llarp/constants/files.hpp>
+#include <llarp/constants/version.hpp>
+#include <llarp/util/buffer.hpp>
+#include <llarp/util/fs.hpp>
+#include <llarp/router_contact.hpp>
+
+#include <sstream>
+
+#ifndef _WIN32
+#include <openssl/x509.h>
+#endif
+
+namespace
+{
+  int
+  exit_with_message(std::string msg, int exitcode)
+  {
+    std::cout << msg << std::endl;
+    return exitcode;
+  }
+}  // namespace
+
+int
+main(int argc, char* argv[])
+{
+  std::string bootstrap_url{"https://seed.lokinet.org/lokinet.signed"};
+  if (argc > 1)
+  {
+    bootstrap_url = argv[1];
+  }
```

Comment:
This should also support passing
"mainnet" or "lokinet" for the mainnet bootstrap
"testnet" for the testnet bootstrap